juriscraper/pacer/reports.py

```diff
@@ -380,3 +380,14 @@ def is_pdf_sealed(self, pacer_case_id, pacer_doc_id, pacer_magic_num=None):
         )
         sealed = "You do not have permission to view this document."
         return sealed in r.content
+
+    def is_entry_sealed(
+        self, pacer_case_id, pacer_doc_id, pacer_magic_num=None
```

Comment:
I would just add types to the arguments.

Reply:
Thanks for the suggestion 👍